credmark/cmf/types/price.py

```diff
@@ -1,16 +1,25 @@
 from typing import (
-    Union
+    Union,
+    List,
+)
+from credmark.dto import (
+    DTO,
+    DTOField,
+    IterableListGenericDTO,
+    PrivateAttr,
 )
-from credmark.dto import DTO, DTOField
 from .token import Token
+from .address import Address
 
 
 class Price(DTO):
     price: Union[float, None] = DTOField(None, description='Value of one Token')
+    src: Union[str, None] = DTOField(None, description='Source')
```

Comment:
this seems like more a debug function than a production function. it's also abbreviated, which we should avoid in general